prow/cmd/jenkins-operator/logs.go

```diff
@@ -56,3 +66,18 @@ func handleLog(jc *jenkins.Client) http.HandlerFunc {
 		}
 	}
 }
+
+func getRealJenkinsLogPath(path string) (string, error) {
+	jobMatches := reJenkinsJobURL.FindStringSubmatch(path)
+	if len(jobMatches) == 0 {
+		return "", errors.New("job name invalid")
```

Comment:
pls make the error more informative, something like "job name does not match <regex>"